### PR TITLE
bug 1914896 - add warning note re security risk due to auto im…

### PR DESCRIPTION
**Pull request by lpettyjo**
…age update disabled by default

https://bugzilla.redhat.com/show_bug.cgi?id=1914896

@agomezpr PTAL.

https://deploy-preview-29298--osdocs.netlify.app/openshift-enterprise/latest/security/container_security/security-understanding.html

### CODE REVIEW

**Comment by lpettyjo** [comment]
Original requester has approved this change.

@xiuwang PTAL.

**Comment by xiuwang** [comment]
@lpettyjo 
LGTM

File: security/container_security/security-understanding.adoc [hunk 1]
```diff
@@ -9,6 +9,14 @@ Securing a containerized application relies on multiple levels of security:
 
 * Container security begins with a trusted base container image and continues
 through the container build process as it moves through your CI/CD pipeline.
++
+[IMPORTANT]
+====
+Image streams by default do not automatically update. This default behavior might create a security issue
+because security updates to images referenced by an image stream will not automatically occur.
```
Comment (codyhoag) [inline security/container_security/security-understanding.adoc:16]: s/will not/do not

File: security/container_security/security-understanding.adoc [hunk 2]
```diff
@@ -9,6 +9,14 @@ Securing a containerized application relies on multiple levels of security:
...
+For information about how to override this default behavior,
+see xref:../../openshift_images/image-streams-manage.adoc#images-imagestreams-import_image-streams-managing[Configuring periodic importing of imagestreamtags].
+====
```
Comment (codyhoag) [inline security/container_security/security-understanding.adoc:19]: Can you remove the hard wraps so the admonition text is all on one line?

**Review by codyhoag — COMMENTED** [review]
Just a couple nits; looks good otherwise!

**Pull request merged**